modules/admin-ui-picard/app/src/components/shared/RegistrationModal.js

```diff
@@ -0,0 +1,514 @@
+                                                           id="adopter_allows_err_reports"
+                                                           className="form-control"/>
+                                                    <label htmlFor="adopter_allows_err_reports">{t('ADOPTER_REGISTRATION.MODAL.FORM_STATE.ERROR_REPORTS')}</label>
+                                                </div>
+                                            </fieldset>
+                                            <fieldset>
+                                                <legend>{t('ADOPTER_REGISTRATION.MODAL.FORM_STATE.POLICY_HEADLINE')}</legend>
+                                                <div className="form-group form-group-checkbox">
+                                                    <Field type="checkbox"
+                                                           name="agreedToPolicy"
+                                                           id="agreedToPolicy"
+                                                           className="form-control"/>
+                                                    <label htmlFor="agreedToPolicy">
+                                                        <span>{t('ADOPTER_REGISTRATION.MODAL.FORM_STATE.READ_TERMS_OF_USE_BEFORE')}</span>
+                                                        <span className="link"
+                                                              onClick={() => setState(states[state].nextState[2])}>
+                                                    {t('ADOPTER_REGISTRATION.MODAL.FORM_STATE.READ_TERMS_OF_USE_LINK')}
+                                                </span>
+                                                        <span>{t('ADOPTER_REGISTRATION.MODAL.FORM_STATE.READ_TERMS_OF_USE_AFTER')}</span>
+                                                    </label>
+                                                </div>
+                                            </fieldset>
+                                        </div>
+                                    </div>
+                                </div>
+                            )}
+
+                            {/* navigation buttons depending on state of modal */}
+                            <footer>
+                                {states[state].buttons.submit && (
+                                    <div className="pull-right">
+                                        {/* submit of form content */}
+                                        {state === "form" ? (
+                                            <button className={cn("submit", {
+                                                active: (formik.isValid && formik.values.agreedToPolicy),
+                                                inactive: !(formik.isValid && formik.values.agreedToPolicy)
+                                            })}
+                                                    disabled={!formik.values.agreedToPolicy}
```

Comment:
Shouldn't it be `disabled=!{formik.isValid && formik.values.agreedToPolicy}` instead of `disabled={!formik.values.agreedToPolicy}`?
```suggestion
                                                    disabled=!{formik.isValid && formik.values.agreedToPolicy}
```